meta-mender-core/recipes-mender/mender-client/mender_5.0.0.bb

```diff
@@ -0,0 +1,41 @@
+require mender-client-cpp.inc
+require mender_4.x.inc
```

Comment:
Should we have a `mender_5.x.inc` and include that instead? Might be a bit overkill since it would be similar to the current one, so feel free to ignore this, I don't have a strong opinion

Reply:
Good observation, thanks.

But no, this `require` is only needed for 4.x recipes because it handles dependencies for the community update modules that we have removed from 5.x.

Reply:
Oh, wait! You actually found a problem! -> the `require` needs to be removed :see_no_evil:

Reply:
Fixed now. Big thanks